src/main/webapp/app/home/home.component.html

```diff
@@ -1,93 +1,95 @@
-<div class="row">
-    <div class="col-md-8 offset-md-2 text-center">
-        <h1 jhiTranslate="home.title">Welcome to Artemis!</h1>
-        <p class="lead" jhiTranslate="home.subtitle">Interactive Learning with Individual Feedback</p>
-        <div *ngIf="!accountName" class="lead" [jhiTranslate]="'home.pleaseSignIn'">Please sign in with your account.</div>
-        <div *ngIf="accountName" class="lead" [jhiTranslate]="'home.pleaseSignInAccount'" [translateValues]="{ account: accountName }">Please sign in with your account.</div>
+<div [hidden]="loading">
```

Comment:
Is the use of `hidden` intentional? Why not use `ngif (hmtl content is not loaded when the statement evaluates to false)?

Reply:
Yes, the content not being loaded was exactly the reason I changed this to `hidden`. I had trouble focusing the input element for some reason, but part of the code has changed so I can retry.